<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DnsTestWidget</class>
 <widget class="QWidget" name="DnsTestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>125</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0">
     <item>
      <widget class="QCheckBox" name="chkTestResultFor">
       <property name="text">
        <string>Test result for</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbTestResultFor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTest">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>24</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="0,2,0,0,0" rowminimumheight="0,0,10,0" columnminimumwidth="0,0,0,0,0">
     <property name="horizontalSpacing">
      <number>2</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblServer">
       <property name="text">
        <string>Server</string>
       </property>
       <property name="buddy">
        <cstring>cmbServer</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cmbRequest">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="lblSecTimeout">
       <property name="text">
        <string>sec</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbProtocol">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>UDP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TCP</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="spinPort">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>53</number>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btnPort">
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../qHostMonitor.qrc">
         <normaloff>:/img/browse.png</normaloff>:/img/browse.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbServer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblRequest">
       <property name="text">
        <string>Request</string>
       </property>
       <property name="buddy">
        <cstring>cmbRequest</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lblRequestType">
         <property name="text">
          <string>Request type</string>
         </property>
         <property name="buddy">
          <cstring>cmbRequestType</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbRequestType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AAAA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CNAME</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MX</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TXT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SPF</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblProtocol">
       <property name="text">
        <string>Protocol</string>
       </property>
       <property name="buddy">
        <cstring>cmbProtocol</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblPort">
       <property name="text">
        <string>Port</string>
       </property>
       <property name="buddy">
        <cstring>spinPort</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="spinTimeout">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <number>600</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lblTimeout">
       <property name="text">
        <string>Timeout</string>
       </property>
       <property name="buddy">
        <cstring>spinTimeout</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="5">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
